src/app/pages/product/product-detail/product-detail.component.html

```diff
@@ -44,6 +44,10 @@ <h1>
       ></ish-lazy-product-add-to-order-template>
 
       <ish-lazy-product-add-to-quote [cssClass]="'btn-secondary btn-block'"></ish-lazy-product-add-to-quote>
+
+      <ish-lazy-product-notification-edit
+        [cssClass]="'btn btn-secondary btn-block'"
```

Comment:
cssClass="btn ..."

Reply:
What di you mean here? Remove the "btn" class?

Reply:
@SGrueber I now understood: change from `[cssClass]="'btn btn-secondary btn-block'"` to  `cssClass="btn btn-secondary btn-block"`